app/controllers/api/v2/instance_hosts_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Api
+  module V2
+    class InstanceHostsController < V2::BaseController
```

Comment:
I don't see a GET implemented for this API controller. Is that implicit?

Reply:
No, I dropped the get endpoint since it didn't provide anything over a scoped search on hosts.

Reply:
Then I wonder how you implemented existence checks but it looks like you retrieve the host itself and the `foreman` facet. I wonder if a listing of all hosts still makes sense though. Feel free to leave it for later, but I can imagine @evgeni would be interested in it as well with Foreman Ansible Modules.

Reply:
I don't think I need to care about listing instance hosts in FAM today… And I can still yell at y'all tomorrow if it will be needed ;-)

(no, seriously, going via the facet seems fine for me?)

Reply:
Seeing if I understand correctly the difference:

 1) Asking instancehosts to show me all the hosts associated to the Foreman instance (this was removed)
 2) Asking the hosts API to return me all hosts that are assigned to the Foreman instance

The first one seems more straight forward and clear with less overhead for someone wanting to get all hosts part of the instance.

Reply:
That sounds right. The first one is indeed more straight forward, but it felt like that by itself is not enough added value to carry yet another api endpoint. If you can imagine it being useful somewhere, I can put it back

Reply:
I tend towards if we know it is an action someone will want to take, whether UI using it or a user debugging, and its a more straight forward solution to have a dedicated API, especially when it completes the CRUD bingo card, than we should go for it. That becomes much easier to understand, remember and construct than a search query to hosts.

Reply:
What information would you like to see there? Originally it used the same template when rendering the response as host index. Would that be fine or would you expect to see something different there?

Reply:
Going with the same template host index uses.